docs/CONNECTOR_PROTOCOL.md

```diff
@@ -125,6 +235,28 @@ In addition to the connector index `.elastic-connectors`, we have an additional
   connector_id: string; -> ID of the connector document in .elastic-connectors
   status: string; -> Same enum as last_sync_status above except the null value
   error: string; -> Optional error message
+  filtering: {          -> Filtering rules
```

Comment:
My 2 cents on whether we should copy the whole connector protocol to job index:
https://docs.google.com/document/d/1E2NPLPbJb6cu_Ir3PznO7AcV1wTLVjh53ma9Q9g4ECc/edit?disco=AAAAi7bhH3U

Reply:
I'm not going to add the whole connector definition as part of this PR. I only need the filtering rules. I'm not opposed to Dmitrii making that change later in this release when he gets to it, if that's the right call then.

Reply:
>I'm not opposed to Dmitrii making that change later in this release when he gets to it, if that's the right call then.

We need to take care of backward compatibility because Crawler is GA.

Even if we want to construct the snapshot, do you think it's better to group them? e.g. 
```
connector: {
  filtering: { ... },
  configuration: { ... },
  ...
}
```